servers/rendering/rasterizer_rd/rasterizer_storage_rd.cpp

```diff
@@ -2827,10 +2825,21 @@ void RasterizerStorageRD::_mesh_surface_generate_version_for_input_mask(Mesh::Su
 				case RS::ARRAY_TEX_UV2: {
 					vd.format = RD::DATA_FORMAT_R32G32_SFLOAT;
 				} break;
+				case RS::ARRAY_CUSTOM0:
```

Comment:
The formats above probably need to be adjusted to reflect the new defaults.

Reply:
those are ok, they are the default values fed to the shader when the array does not contain them.